### DISCUSSION TITLE
java.lang.NullPointerException

### DISCUSSION THREAD

**Discussion by vmotakat** [GRIDAPPSD/gridappsd-forum#144] [API] [answered]
Hi,
I am trying to run GridAPPS-D Ocher with recent edits. I am loading master spreadsheet but simulation getting error as bellow
<img width="1241" alt="Screen Shot 2022-09-08 at 9 03 38 AM" src="https://user-images.githubusercontent.com/102381298/189188639-5254f0ba-112e-4139-86d6-f3a3ebe467d9.png">

Can you help me to fix this error
@poorva1209

**Comment by AAndersn** [comment, marked as answer]
This problem is caused by modifying the .xlsx file in Ubuntu LibreOffice instead of MS Excel. LibreOffice modifies the file formatting using its own ODF spec, after which the file is unreadable by the GOSS java code. Re-saving the file using MS Excel resolves the issue.
